adam-cli/pom.xml

```diff
@@ -14,22 +14,35 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>appassembler-maven-plugin</artifactId>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
         <configuration>
-          <programs>
-            <program>
-              <mainClass>org.bdgenomics.adam.cli.ADAMMain</mainClass>
-              <id>adam</id>
-            </program>
-          </programs>
+          <createDependencyReducedPom>true</createDependencyReducedPom>
+          <filters>
+            <filter>
+              <artifact>*:*</artifact>
+              <excludes>
+                <exclude>META-INF/*.SF</exclude>
+                <exclude>META-INF/*.DSA</exclude>
+                <exclude>META-INF/*.RSA</exclude>
+              </excludes>
+            </filter>
+          </filters>
```

Comment:
>> test assembly artifact in build for downstream applications

> I don't think this change should impact downstream applications, except possibly plugins. Are plugins your concern?

avocado, rice, qc-metrics, etc. all use classpath-determining scripts similar to those being removed here in their ```bin``` directories.